js/src/spbc-settings_tab--settings_general.js

```diff
@@ -13,4 +13,32 @@ jQuery(document).ready(function(){
 		jQuery('#spbc_admin_timezone').val(d.getTimezoneOffset()/60*(-1));	
 	}
 	
+	// Function for shuffle authentication unique keys and salts
+	function spbcShuffleSalts() {
```

Comment:
It's just a comment. No need to make a patch here.

Please, in future, use spbc_sendAJAXRequest() to perform AJAX.